docs/challenge/updates.md

```diff
@@ -1,58 +1,90 @@
-On this page, we provide weekly updates regarding the first BEHAVIOR Challenge, including important bug fixes, announcements of new features, and clarifications regarding challenge rules.
+# BEHAVIOR Challenge Weekly Updates
 
+On this page, we provide weekly updates regarding the first **BEHAVIOR Challenge**, including important bug fixes, new feature announcements, and clarifications about challenge rules.
 
+---
 
-### 09/28/2025
+### 10/06/2025
 
 **Challenge rule clarifications:**
 
-1. There are no formal registration required to participate the challenge, feel free to submit your result directly if you have one!
+1. During evaluation, only the task-relevant object poses and the robot’s initial pose will change.  
```

Comment:
Fix this: "change" here seems to suggest that the other objects will be fixed, which is not true. You mean only these things will _be randomized._